tests/test.py

```diff
@@ -547,6 +548,85 @@ def test_spi_sim_transfer(self):
+        
+    def test_gpio_set_bus_voltage(self):
+        if self.use_simulator:
+            self.skipTest("For real device only")
+
+        self.device.open()
+        
+        gpio = self.device.create_interface("gpio")
+        
+        (success, result) = gpio.set_bus_voltage(3300)
+        
+        self.assertTupleEqual((success, result), (True, 3300))
+        self.assertEqual(gpio.bus_voltage, 3300)
+        
+        self.device.close()
+
+    def test_gpio_configure_pin(self):
+        if self.use_simulator:
+            self.skipTest("For real device only")
+
+        self.device.open()
+        
+        gpio = self.device.create_interface("gpio")
+        
+        (success, result) = gpio.configure_pin(GpioPinNumber.GPIO_6, GpioFunctionality.DIGITAL_OUTPUT)
+        self.assertEqual(success, True)
+        self.assertEqual(result, "Success")
+        
+        (success, result) = gpio.configure_pin(GpioPinNumber.GPIO_5, GpioFunctionality.DIGITAL_INPUT)
+        self.assertEqual(success, True)
+        self.assertEqual(result, "Success")
+        
+        self.device.close()
+
+    def test_gpio_digital_write_read(self):
+        if self.use_simulator:
+            self.skipTest("For real device only")
+
+        self.device.open()
+        
+        gpio = self.device.create_interface("gpio")
+        
+        gpio.configure_pin(GpioPinNumber.GPIO_6, GpioFunctionality.DIGITAL_OUTPUT)
+        gpio.configure_pin(GpioPinNumber.GPIO_5, GpioFunctionality.DIGITAL_INPUT)
+        
+        (success, _) = gpio.digital_write(GpioPinNumber.GPIO_6, GpioLogicLevel.HIGH)
+        self.assertEqual(success, True)
+        
+        (success, value) = gpio.digital_read(GpioPinNumber.GPIO_5)
+        self.assertEqual(success, True)
+        self.assertEqual(value, GpioLogicLevel.HIGH.name)
+        
+        (success, _) = gpio.digital_write(GpioPinNumber.GPIO_6, GpioLogicLevel.LOW)
+        self.assertEqual(success, True)
+        
+        (success, value) = gpio.digital_read(GpioPinNumber.GPIO_5)
+        self.assertEqual(success, True)
+        self.assertEqual(value, GpioLogicLevel.LOW.name)
+        
+        self.device.close()
+
+    def test_gpio_set_disable_interrupt(self):
+        if self.use_simulator:
+            self.skipTest("For real device only")
+        self.device.open()
+        
+        gpio = self.device.create_interface("gpio")
+        
+        gpio.configure_pin(GpioPinNumber.GPIO_5, GpioFunctionality.DIGITAL_INPUT)
+        
+        (success, result) = gpio.set_interrupt(GpioPinNumber.GPIO_5, GpioTriggerType.TRIGGER_BOTH_EDGES)
+        self.assertEqual(success, True)
+        self.assertEqual(result, "Success")
+        
+        (success, result) = gpio.disable_interrupt(GpioPinNumber.GPIO_5)
+        self.assertEqual(success, True)
+        self.assertEqual(result, "Success")
+        
+        self.device.close()
 
     def test_target_reset_read_reset_action(self):
```

Comment:
why is this here?

Reply:
In that part we are testing that `gpio.disable_interrupt()` works as expected. That answers your question?